osgar/drivers/kloubak.py

```diff
@@ -245,7 +259,23 @@ def process_packet(self, packet, verbose=False):
 
     def slot_can(self, timestamp, data):
         self.time = timestamp
-        limit_l, limit_r = compute_desired_erpm(self.desired_speed, self.desired_angular_speed)
+        
+        angel_diff = None
```

Comment:
You probably meant `angle` here, right? Anyway, this line could be deleted.

Reply:
fixed